src/mixins/behaviors.js

```diff
@@ -81,7 +85,7 @@ export default {
     _invoke(this._behaviors, 'undelegateEntityEvents');
   },
 
-  _destroyBehaviors(args) {
+  _destroyBehaviors(...args) {
```

Comment:
destructuring function arguments does not cause IE to break because it can assume that `arguments` is already array-like.  But destructuring any variable requires the babel helper we can't use.  FWIW this function is private and only called in one location, so it shouldn't be an issue.
If we wanted to keep the arguments the same for some reason we'd have to create a `_.partial` function and then apply the arguments to it... that seems worse.